axelor-human-resource/src/main/java/com/axelor/apps/hr/service/allocation/AllocationLineComputeServiceImpl.java

```diff
@@ -216,7 +285,6 @@ protected BigDecimal computeProrata(
   @Override
   public BigDecimal getAllocatedTime(Project project, Sprint sprint) {
     List<AllocationLine> allocationLineList = allocationLineRepo.findByProject(project).fetch();
-
```

Comment:
please avoid modifying whitespace on unrelated files/methods, it makes the change more difficult to understand and can cause merge conflicts for nothing